subprojects/android-test/test-app/src/androidTest/kotlin/com/avito/android/ui/test/OverlapTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.avito.android.ui.test
+
+import com.avito.android.ui.OverlapActivity
+import org.junit.Rule
+import org.junit.Test
+
+class OverlapTest {
+
+    @get:Rule
+    val rule = screenRule<OverlapActivity>()
+
+    @Test
+    fun isOverlapped_button_under_snackbar() {
+        rule.launchActivity(null)
+
+        Screen.overlapScreen.snackButton.click()
+
+        Screen.overlapScreen.snackButton.checks.isOverlapped()
```

Comment:
I think we need only IsDisplayedWithoutOverlapped
This scenario we should check by occurred exception

Reply:
Could you please explain why? In my opinion, it's a separate check like `isDisplay` or `isVisible`. In task MBS-8512 (internal) they will be combined into one method.

Reply:
Because users need to check not overlapped scenario. Why do they check `isOverlapped`?

Reply:
I think that API should have two related methods: `is*` and `isNot*`